tests/test_widgets.py

```diff
@@ -185,6 +185,34 @@ class W(SuffixedMultiWidget):
         }, {}, 'price')
         self.assertEqual(result, ['1', 'lt'])
 
+    def test_value_omitted_from_data(self):
+        class A(SuffixedMultiWidget):
+            suffixes = ['_b']
+
+        a = A(widgets=[BooleanWidget])
+
+        result = a.value_omitted_from_data(['a', 'b', 'c'], './.', 'test')
```

Comment:
I'm not sure about why `['a', 'b', 'c'], './.'` was chosen for `data, files` here. Would simpler `None, []` or so do as well?